graphics_magick-cleared-console-20191213.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "xpos": 114,
+      "ypos": 95,
+      "width": 809,
+      "height": 524,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "cleared-console"
+  ]
+}
```

Comment:
I don't see where this needle should be needed. It has a generic tag "cleared-console" but with a filename stating it would be explicit for "graphics_magick". I know, the openQA needle editor suggests this filename but we should not use that for generic tags. Maybe you are missing an update to needles from the main repo?

Reply:
Maybe this is the cause. While I was creating the needles this suggested me this needle. Checking...

Reply:
Done